recipes/ligo-scald/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "ligo-scald" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "88b9ff3b881a381dbd62467b37bbc55fd88b6e999e33e1a1b8b6d3c28952a094" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+
+  run:
+    - python
+    - six
+    - numpy>=1.7.1
+    - scipy
+    - h5py>=2.3.0
+    - python-lal  # [not win]
+    - pyyaml
+    - bottle
+    - ligo-common
+    - ligo-segments
```

Comment:
I think you require` setuptools` at run time to provide `pkg_resources`, right?

Reply:
Good catch, I added this.